src/components/FeatureInformation/FeatureInformation.scss

```diff
@@ -25,7 +25,17 @@
   }
 
   .wkp-feature-information-body {
-    padding: 10px;
+    & > * {
```

Comment:
I have changed/fixed the popup styling (core, construction, and infofpw), see commits 7e716dc09cae2c34fc18f2fde9f9a7b4c85511cf and feab723166c2b24ff5e4c4c6148d808d826c0d94